.travis.yml

```diff
@@ -2,27 +2,25 @@ dist: xenial
 language: python
 python:
 - '2.7'
-- '3.4'
 - '3.5'
 - '3.6'
 - '3.7'
-- 3.7-dev
+- '3.8'
 - 3.8-dev
 - nightly
 - pypy
 - pypy3
+addons:
+  apt:
+    packages:
+    - pandoc
 install:
 - travis_retry pip install -r requirements.txt
 - travis_retry pip install coveralls
 script:
 - coverage run --source=jsonpointer tests.py
 after_script:
 - coveralls
-sudo: false
-addons:
-  apt:
-    packages:
-    - pandoc
```

Comment:
Any particular reason why you moved that?, and removed the `sudo: false` ? I'd prefer to keep the diffs clean of unrelated changes.

Reply:
Travis CI have completely deprecated the __sudo:__ tag.  The __sudo__ command is _always_ available in Travis and there is no longer anything that you can do to turn it off.

The move keeps things in chronological order because the apt tasks happen before the install tasks in the Travis CI logs.